modules/ROOT/pages/major-changes.adoc

```diff
@@ -0,0 +1,144 @@
+= Major Changes in Fedora CoreOS
+:toc:
+
+This is a list of major changes that were introduced in Fedora CoreOS with the
+notes associated with them.
+Those changes are also announced on the https://lists.fedoraproject.org/archives/list/coreos-status@lists.fedoraproject.org/[coreos-status mailing list].
+Note that this is not an exhaustive list of changes in Fedora CoreOS and only includes major changes that may require manual actions.
+This list is in reverse chronological order to keep recent changes at the top.
+
+////
+// Template for major changes:
+
+== Name
+
+=== Planning
+
+This change will be rolled out ...
+
+|===
+|Update Stream |Targeted release date
+
+|`next` | ...
+|`testing`| ...
+|`stable`| Will follow `testing` as usual
+|===
+
+=== Notes
+
+...
+////
+
+== Podman v4.0
```

Comment:
Let's make this a hyperlink, or clearly link to https://fedoraproject.org/wiki/Changes/Podman4.0